src/test/java/com/github/junrar/ArchiveTest.java

```diff
@@ -62,6 +83,49 @@ public void testTikaDocs() throws Exception {
         }
     }
 
+    /*
+     The file is shifted by 1-hour because it was created in Europe/Amsterdam.
+     Times in a RAR file are stored in the MS-DOS style, so the fields are always the same but the resulting timestamp is not.
+
+     Original timestamps:
+     MTime: 2022-02-23T09:24:19.191543300Z
+     CTime: 2022-02-23T09:34:59.759754700Z
+     ATime: 2022-03-02T17:45:18.694091100Z
+
+     Ensure the fields remain constant across timezones.
+     */
+    @ParameterizedTest
+    @ValueSource(strings = {"America/Los_Angeles", "America/Sao_Paulo", "Europe/Amsterdam", "Asia/Kolkata", "default"})
+    public void testArchiveExtTimes(String timeZone) throws Exception {
+        TimeZone defaultTimezone = TimeZone.getDefault();
+        if (!"default".equals(timeZone)) {
+            TimeZone.setDefault(TimeZone.getTimeZone(timeZone));
```

Comment:
Wouldn't this change the behaviour for subsequent tests also ?

Reply:
That's what line 126 is for.
As long as we're not running the tests in parallel, it should be fine.

I can also add the TimeZone to the constructor of Archive or try to pass it some other way if you think that's better.

Reply:
Maybe we could use https://junit-pioneer.org/docs/default-locale-timezone/#defaulttimezone

Reply:
Sure. I can try that.

Reply:
Line 126 would not always be triggered if you have an exception, no?

Reply:
Yes, but if you have an exception the test fails anyway and the build stops, unless we're running the tests in the suite in parallel.

Reply:
better plan for the future if we can!

Reply:
I have to check how that solution works because it might not be different than a try-finally block, which won't solve parallel test execution

Reply:
@gotson done :)